From 47bd06bd29e2e7856e175e570e2aab5bd0b8030b Mon Sep 17 00:00:00 2001
From: Jianxu Zheng <jian.xu.zheng@intel.com>
Date: Wed, 20 Aug 2014 12:15:04 +0800
Subject: [PATCH 146/164] atomisp.h: add depth camera preview pause event

Now camera hal receive pause and resume command from application,
but it is not accuracy for frame sync. So we add one event to pause
preview buffer queue when take picture.
It will be much more accuracy for depth capturing.
 1. Add one event;
 2. start filling preview buffer queue;
 3. release all buffers in preview buffer queue after target preview
    buffer is taken for capturing.

Change-Id: I07f077d448273fc15dc8bb25154103ae95a19928
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-8033
Signed-off-by: Jianxu Zheng <jian.xu.zheng@intel.com>
---
 include/linux/atomisp.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/atomisp.h b/include/linux/atomisp.h
index 545a508..4004315 100644
--- a/include/linux/atomisp.h
+++ b/include/linux/atomisp.h
@@ -1319,6 +1319,7 @@ struct atomisp_sensor_ae_bracketing_lut {
 #define V4L2_EVENT_ATOMISP_METADATA_READY   (V4L2_EVENT_PRIVATE_START + 2)
 #define V4L2_EVENT_ATOMISP_RAW_BUFFERS_ALLOC_DONE   (V4L2_EVENT_PRIVATE_START + 3)
 #define V4L2_EVENT_ATOMISP_ACC_COMPLETE     (V4L2_EVENT_PRIVATE_START + 4)
+#define V4L2_EVENT_ATOMISP_PAUSE_BUFFER	    (V4L2_EVENT_PRIVATE_START + 5)
 
 /* Nonstandard color effects for V4L2_CID_COLORFX */
 enum {
-- 
1.7.9.5

